notebooks/official/pipelines/control_flow_kfp.ipynb

```diff
@@ -127,6 +127,56 @@
         "6. Open this notebook in the Jupyter Notebook Dashboard.\n"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        template_path="control_pipeline.json".replace(" ", "_"),
</pre><p>I know this is due to the auto-assembler, but this is awkward code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>